dev-docs/workflows/build-develop-deploy.md

```diff
@@ -0,0 +1,178 @@
+# Getting started locally
+
+The following are instructions for building all components in the constellation repository, except for images. A manual on how to build images locally can be found in the [image README](/image/README.md).
+
+Prerequisites:
+
+* 20GB (minimum), better 40 GB disk space (required if you want to cross compile for all platforms)
+* [Latest version of Go](https://go.dev/doc/install).
+* [Bazelisk installed as `bazel` in your path](https://github.com/bazelbuild/bazelisk/releases).
+* [Docker](https://docs.docker.com/engine/install/). Can be installed with these commands on Ubuntu 22.04: `sudo apt update && sudo apt install docker.io`. As the build spawns docker containers your user account either needs to be in the `docker` group (Add with `sudo usermod -a -G docker $USER`) or you have to run builds with `sudo`. When using `sudo` remember that your root user might (depending on your distro and local config) not have the go binary in it's PATH. The current PATH can be forwarded to the root env with `sudo env PATH=$PATH <cmd>`.
+
+## Prequisites
+
+### Linux
+
+* Packages on Ubuntu:
+
+  ```sh
+  sudo apt install build-essential cmake libssl-dev pkg-config libcryptsetup12 libcryptsetup-dev
+  ```
+
+* Packages on Fedora:
+
+  ```sh
+  sudo dnf install @development-tools pkg-config cmake openssl-devel cryptsetup-libs cryptsetup-devel
+  ```
+
+### Mac
+
+```sh
+brew install bash
+```
+
+to fix unsupported shell options used in some build script.
+
+To troubleshoot potential problems with bazel on ARM architecture when running it for the first time, it might help to purge and retry:
+
+```sh
+bazel clean --expunge
+```
+
+## Settings
+
+### Set build version
+
+There exists an `oss` and an `enterprise` version for licensing reasons.
+
+`enterprise`: recommended, complete feature set but some features are not licensed under Open-Source
+
+`oss`: default when building from the OSS repo, but non OSS-licensed features are missing (e.g. verify boot measurements). The [official docs](https://docs.edgeless.systems/constellation/) are written to be compatible with `oss`.
```

Comment:
```suggestion
`oss`: default when building from the OSS repo, but non OSS-licensed features are missing (e.g. fetching expected measurements). The [official docs](https://docs.edgeless.systems/constellation/) are written to be compatible with `oss`.
```
Verifying boot measurements should always be supported, since oss/enterprise feature gates only influence the behavior of the CLI, not the whole Constellation.

Also not sure about the docs, I think its the other way around? Maybe we should discuss during standup